src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -3,20 +3,29 @@ title: 'Running large tests'
 excerpt: 'How to run large-scale k6 tests without distributed-execution'
 ---
 
-This document explains how to launch a large-scale k6 test on a single machine without the need for distributed execution.
+A common misconception about load testing is that it requires [distributed execution](#distributed-execution) (that is, load generated from multiple machines).
+This isn't the case with k6.
```

Comment:
This is a very strong statement that arguably is wrong. 

Can we have not so marketing message of 
```suggestion
A common misconception about load testing is that it requires [distributed execution](#distributed-execution) (that is, load generated from multiple machines).
This isn't necessarily the case with k6.
```

Reply:
I support changing the wording here and in other places. 

Haven't reviewed it. @mstoykov, it can help if you can point other sentences.

Reply:
Hhmm what would you say is wrong about it? That it's not a common misconception, or that k6 does require distributed execution?

I'm not sure how common of a misconception it is, but k6 certainly doesn't _require_ it. I'd say there's some truth in both statements.

I can add necessarily to soften this, but maybe we should rephrase this altogether? WDYT @MattDodsonEnglish?

Reply:
How about this?

> k6 can generate a lot of load from a small machine.
> With proper monitoring and script optimization, you might be able to run a rather large load test without needing distributed execution.

Reply:
Thanks, Matt!

I changed this, and a few other places that were too harsh.

@mstoykov Let us know if 62bd7e8d is an improvement.

Reply:
For me that changes is great. 

IMO it was wrong on a lot of levels but especially on the hardness of "it isn't the case".

But distributed mode isn't just used for *scale* it also used to make the test come from multiple (usually) geo distributed machines. 

And we are not doing some very complex test - for something more complex we will need even more resources.

Sometimes there are other considerations (as the article already goes through) such as the number of ports and the need for multiple IPs. 

Arguably those are fixable *without* distributed execution, but probably only to an extend.

So the whole hardness of the "it isn't the case" is the *biggest* problem.